packages/reports/addon/components/filter-builders/base.ts

```diff
@@ -74,4 +74,15 @@ export default class BaseFilterBuilder<T extends BaseFilterBuilderArgs = BaseFil
       parameters: { ...this.args.filter.parameters, [key]: value },
     });
   }
+
+  @computed('args.filter.columnMetadata.name')
+  get columnName() {
+    return this.args.filter.columnMetadata.name;
+  }
+
+  @computed('args.filter.parameters')
+  get paramValues() {
+    const params = omit(this.args.filter.parameters || {}, 'as');
+    return Object.values(params);
```

Comment:
We no longer need to omit the `as` parameter (legacy v1 thing)
```suggestion
    return Object.values(this.args.filter.parameters);
```